contracts/eosio.system/src/voting.cpp

```diff
@@ -145,6 +151,9 @@ namespace eosiosystem {
 
       if( set_proposed_producers( producers ) >= 0 ) {
          _gstate.last_producer_schedule_size = static_cast<decltype(_gstate.last_producer_schedule_size)>( top_producers.size() );
+
+         /// @todo Update reward_info::status. It's a complicated task because we don't know exactly when the 
```

Comment:
Maybe we should just update the producer status as part of the voting function here: https://github.com/worldwide-asset-exchange/wax-system-contracts/blob/standby-producers/contracts/eosio.system/src/voting.cpp#L213?? At the very end of it we could determine which producers are in the top 21 versus the next 36 and set the status at that point?